src/frontend/Dockerfile

```diff
@@ -1,38 +1,28 @@
-FROM node:18-alpine AS deps
+FROM node:18-alpine AS build
```

Comment:
TBH I would create a new `Dockerfile` if it's actually needed (AFAIK it doesn't seem so 🤔 ). This would make it easier to keep the `grafana` branch up-to-date with the `main` branch from the upstream but also it won't affect building the image in the original context.